src/Sarif.Multitool.FunctionalTests/Sarif.Multitool.FunctionalTests.csproj

```diff
@@ -64,4 +63,6 @@
     <PackageReference Include="xunit.runner.visualstudio" Version="2.3.1" />
     <DotNetCliToolReference Include="dotnet-xunit" Version="2.3.1" />
   </ItemGroup>
+
+  <ProjectExtensions><VisualStudio><UserProperties TestData_4EndColumnMustNotBeLessThanStartColumn_4EndColumnEqualsStartColumnNoEndLine_1sarif__JSONSchema="http://json.schemastore.org/sarif-2.0.0" TestData_4EndColumnMustNotBeLessThanStartColumn_4EndColumnLessThanStartColumnInRelatedLocation_1sarif__JSONSchema="http://json.schemastore.org/sarif-2.0.0" TestData_4EndColumnMustNotBeLessThanStartColumn_4EndColumnGreaterThanStartColumn_1sarif__JSONSchema="http://json.schemastore.org/sarif-2.0.0" TestData_4EndColumnMustNotBeLessThanStartColumn_4EndColumnEqualsStartColumn_1sarif__JSONSchema="http://json.schemastore.org/sarif-2.0.0" /></VisualStudio></ProjectExtensions>
```

Comment:
??? I think VS added this when I edited the SARIF files and typed "sarif-2.0.0" into the Schema field at the top of the editor. I'm going to remove this line.